### DISCUSSION TITLE
[Help] Price Rules equivalent for Subscription Pricing

### DISCUSSION THREAD

**Discussion by igoraugustynski** [swellstores/community#273] [Help] [answered]
Hello,
In my use case I have a group which I intend to use to target all customers who bought a specific product. 
When a customer is in this group, I want to be able to offer him lower prices for some other products. 
With "Standard Pricing" I can achieve this by setting up a Price Rule and selecting the group as condition. This works fine - in my custom FE I get both the old and the new, discounted, price so I can provide a nice UX for the customer by saying "Hey, You're paying X instead of Y because You're a member of group Z".
My issue is that I have some other products which are in fact subscriptions, and I need to be able to change the price of those subscriptions based on the user group too, but there is no such option as "Price Rules" for subscriptions. 
I've read the documentation and found out that Promotions can be used to do this, but the issue is that they are only applied to the cart, not the product itself. This would be fine but I need a way to show the discounted price for logged in customers as soon as possible - on the product card or product detail page. 
Is there a way/workaround to achieve this?

**Comment by ericingram** [comment]
You’re right and this is considered a feature gap currently. A workaround could be to add a custom field to display the discount on subscription products, which I know isn’t ideal.

We still need a proper solution

**Reply by igoraugustynski** [reply]
Is there a way to know if a user is in a given group using swell-js? I could add a custom field but AFAIK there is no way to show/hide it based on groups membership of the user. 
Also, are You guys planning on releasing this feature anytime soon? Is it on the roadmap? I think the best solution would be to mimic the standard pricing way which is great.

**Reply by ericingram** [reply, marked as answer]
I would make a Collection field and specify group + price

You should be able to access user group using swell-js (or graphql for that matter)

It's not currently on the roadmap
